packages/react/src/components/TagButton/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { HTMLProps } from 'react';
+import Icon, { IconType } from '../Icon';
+import { ContentNode } from '../../types';
+import Tag, { TagLabel } from '../Tag';
+import classNames from 'classnames';
+import Button from '../Button';
+
+interface TagButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
+  label: ContentNode;
+  value: ContentNode;
+  icon: IconType;
+  action: (e: React.MouseEvent<HTMLButtonElement>) => void;
+  disabled?: boolean;
+}
+
+const TagButton = ({
```

Comment:
This needs to follow our [component standards](https://cauldron.dequelabs.com/component-guidelines#component-standards) and is missing: 

- `React.forwardRef`